tests/Doctrine/Tests/DBAL/Functional/BlobTest.php

```diff
@@ -55,6 +57,43 @@ public function testInsert()
         self::assertEquals(1, $ret);
     }
 
+    public function testInsertProcessesStream()
+    {
+        $this->_conn->insert('blob_table', [
+            'id'          => 1,
+            'clobfield'   => 'ignored',
+            'blobfield'   => fopen('data://text/plain,test', 'r'),
+            'binaryfield' => fopen('data://text/plain,test', 'r'),
+        ], [
+            ParameterType::INTEGER,
+            ParameterType::STRING,
+            ParameterType::LARGE_OBJECT,
+            ParameterType::LARGE_OBJECT,
+        ]);
+
+        $this->assertBlobContains('test');
+        $this->assertBinaryContains('test');
+    }
+
+    public function testInsertCanHandleStreamLongerThanChunkSize()
```

Comment:
Wouldn't one test be enough to test a large-ish value? If a large value can be stored, a small one can too.